runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -556,6 +559,8 @@ private void handleStartCheckPoint(long address, ILogData logData, UUID streamId
                     .setStartAddress(startAddress)
                     .setStarted(true));
 
+        } catch (InterruptedException ie) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)